src/Controller/User/UserSubmitController.php

```diff
@@ -147,4 +147,36 @@ public function rejectedSubmits(): Response
             'submits' => $this->submitRepository->findUserSubmitsByStatus($this->getUser(), Submit::STATUS_REJECTED),
         ]);
     }
+
+    #[IsGranted('SUBMIT_ACTION', 'submit')]
```

Comment:
Are you sure you should pass a string? Does it work?

Reply:
Yes it does, and this is the only way I found, I could invest further however

Reply:
ok ok, 👍🏼 no need to dig